com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKPokeReticleVisual.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Unity.Profiling;
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public class MRTKPokeReticleVisual : BaseReticleVisual
+    {
+        [SerializeField]
+        [Tooltip("The interactor which this visual represents.")]
+        private PokeInteractor pokeInteractor;
+
+        [SerializeField]
+        [Tooltip("The GameObject which holds the proximity light for the reticle")]
+        private GameObject proximityLight;
+
+        protected void OnEnable()
+        {
+            Application.onBeforeRender += UpdateReticle;
+        }
+
+        protected void OnDisable()
+        {
+            UpdateReticle();
+            Application.onBeforeRender -= UpdateReticle;
+        }
+
+        private static readonly ProfilerMarker UpdateVisualsPerfMarker =
+            new ProfilerMarker("[MRTK] MRTKPokeReticleVisual.UpdateReticle");
+
+        [BeforeRenderOrder(XRInteractionUpdateOrder.k_BeforeRenderLineVisual)]
+        private void UpdateReticle()
+        {
+            using (UpdateVisualsPerfMarker.Auto())
+            {
+                reticle.SetActive(pokeInteractor.enabled && pokeInteractor.isHoverActive);
+
+                // TODO: Ideally we'd want the ReticleVisualScript to be responsible for all aspects of a visuals appearance.
+                // This is because it would allow us to freely use different reticle icons without changing other behavior.
+                // However, until the ReticleMagnetism class is revisited and potentially incorporated into this class, we'll
+                // leave all pose related responsibilities to the ReticleMagnetism component.
```

Comment:
Interesting! Was this potentially fighting with `ReticleMagnetism` in the past and we just didn't notice?

Reply:
Oh sorry, I guess "this" in this case is the copied, deleted, and now commented-out code that was previously controlling this! 
![image](https://user-images.githubusercontent.com/3580640/192067949-b5eaabd1-a270-40ac-83ea-ba2eb61254a5.png)

![image](https://user-images.githubusercontent.com/3580640/192067967-a232fa1d-b1b5-41a4-a392-4c873f0b8fce.png)

Reply:
Oh yeah! It was definitely fighting with reticle magnetism on the orientation side of things, I think positioning though there wasn't any noticeable conflict. All in all, it wasn't super visible before, but I want to consolidate things more onto the top level "ReticleVisual" side rather than on each individual reticle instance.